sdk/storage/azure-storage-file-datalake/MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include *.md
+include azure/__init__.py
+include azure/storage/__init__.py
+include azure/storage/file/__init__.py
+include LICENSE.txt
+recursive-include tests *.py *.yaml
```

Comment:
Do you want to add `recursive-include samples *.py` and put samples in a different folder when you add them.

Reply:
`recursive-include samples *.py *.md` to include the samples README.md  :)